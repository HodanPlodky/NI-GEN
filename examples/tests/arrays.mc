int main() {
    int arr[3];
    int arr2[10];
    arr[0] = 1;
    arr[1] = 20;
    arr[2] = arr[0] + arr[1];
    arr2[2] = 39;
    return arr[2] + arr2[2];
}
