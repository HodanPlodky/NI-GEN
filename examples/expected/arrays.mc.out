input file : examples/tests/arrays.mc
return 60
